kvdb/postgres/config.go

```diff
@@ -4,6 +4,7 @@ import "time"
 
 // Config holds postgres configuration data.
 type Config struct {
-	Dsn     string        `long:"dsn" description:"Database connection string."`
-	Timeout time.Duration `long:"timeout" description:"Database connection timeout. Set to zero to disable."`
+	Dsn            string        `long:"dsn" description:"Database connection string."`
+	Timeout        time.Duration `long:"timeout" description:"Database connection timeout. Set to zero to disable."`
+	MaxConnections int           `long:"maxconnections" description:"The maximum number of open connections to the database."`
```

Comment:
nit: I think it's worth mentioning that the default 0 means unlimited.

Reply:
Added. Also added the implementation for not setting the limit when the value is zero.